packages/vinxi/lib/manifest/dev-server-manifest.js

```diff
@@ -173,7 +173,7 @@ export function createDevManifest(app) {
 											];
 										},
 										output: {
-											path: absolutePath,
+											path: join(router.base, "@fs", absolutePath),
```

Comment:
Not sure on this one

Reply:
this change will not work, on the server we don't want to use `@fs` links. those are for the client. this is being used with the server actions setup. I should add a test for that